src/index.html

```diff
@@ -5,6 +5,7 @@
 
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href ="styles.css">
```

Comment:
Remove this in favor of a `main.scss` which is then imported into your `app.component.scss`.